### ISSUE TITLE
debug: false has no effect

### ISSUE THREAD

**Issue by erlendhade** [experiencor/keras-yolo3#107]
Hi

I am trying to train a detector for six classes, but during training loads of unnecessary data is being outputted to the terminal. 
Example: 

    loss xy, wh, conf, class: 	[36][0.264708728][1.66123188][12.6610851][4.70888519]
    avg_obj 		[9][0.879793286]
    avg_noobj 		[9][0.00372675597]
    avg_iou 		[9][0.569143951]
    avg_cat 		[9][0.999000967]
    recall50 	[9][0.999000967]
    recall75 	[9][0]
    count 	[9][1]
    loss xy, wh, conf, class: 	[9][0.0315124914][0.467090517][0.356625289][0.383489907]
    avg_obj 		[18][0.994455695]
    avg_noobj 		[18][0.00139690912]
    avg_iou 		[18][0.701696754]
    avg_cat 		[18][0.999000967]
    recall50 	[18][0.999000967]
    recall75 	[18][0]
    count 	[18][1]
    loss xy, wh, conf, class: 	[18][0.0805066824][0.342962801][1.43715227][0.508926153]
    avg_obj 		[36][0.752841711]
    avg_noobj 		[36][0.00284046889]
    avg_iou 		[36][0.58843869]
    avg_cat 		[36][0.699929953]

I have set debug to false in the config.json file, but it seems to have no effect. 
What I would like to output is loss and validation loss

thanks in advance for any help

**Comment by AlexTitovWork** [comment]
Hi guys, can you explain these values? I am training a car detector with this project.
loss xy, wh, conf, class: 	[52][0][0][0.154005572][0]
avg_obj 		[52][7.74086232e-07]
avg_noobj 		[52][0.000469937542]
avg_iou 		[52][-nan]
avg_cat 		[52][0.999000967]
recall50 	[52][0]
recall75 	[52][0]
count 	[52][1]
loss xy, wh, conf, class: 	[52][0.482120395][0.0839948431][27.8332672][0]
